lib/canvas/live_events.rb

```diff
@@ -93,6 +93,19 @@ def self.get_submission_data(submission)
     }
   end
 
+  def self.get_attachment_data(attachment)
+    {
+      attachment_id: attachment.global_id,
```

Comment:
We will as it might've been moved by the time we can process it. I'll add it in regardless as it might be useful to other people